bookkeeper-server/src/main/java/org/apache/bookkeeper/http/DecommissionService.java

```diff
@@ -82,14 +86,13 @@ public void run() {
                                 LOG.error("Error handling decommissionBookie: {} with exception {}", bookieSrc, e);
```

Comment:
It would be good to have logging before and after the call to decommissionBookie, so that an admin can see whether the process started and ended if anything goes wrong.